package.json

```diff
@@ -8,9 +8,17 @@
   "engines": {
     "atom": ">=0.116.0"
```

Comment:
Please bump atom version to 0.177.0 or whatever version you are using. It's definitely not 0.116.0. I believe apm just recently implemented checking this version, but it's a good practice nonetheless.